nucliadb/nucliadb/search/search/query.py

```diff
@@ -178,6 +178,11 @@ async def paragraph_query_to_pb(
     request = ParagraphSearchRequest()
     request.reload = reload
     request.with_duplicates = with_duplicates
+
+    # We need to ask for all and cut later
+    request.page_number = 0
+    request.result_per_page = page_number * page_size + page_size
+
     if SearchOptions.PARAGRAPH in features:
```

Comment:
Does it make sense to query the resource search and not have PARAGRAPH as requested feature?

Reply:
Yes, you can search by dates for example (although it's not implemented either (I'll add it now))